web/src/components/bit.dev/toggle-switch.tsx

```diff
@@ -0,0 +1,46 @@
+type ToggleSwitchProps = {
+  day: string
+  closed: boolean
+  setClosed: React.Dispatch<React.SetStateAction<boolean>>
+}
+const ToggleSwitch = ({ day, closed, setClosed }: ToggleSwitchProps) => {
+  return (
+    <>
+      <div className="flex items-center justify-start w-24">
+        <label
+          htmlFor={`open-${day}`}
+          className="flex items-center cursor-pointer"
+        >
+          <div className="relative">
+            <input
+              id={`open-${day}`}
+              type="checkbox"
+              className="sr-only"
+              checked={closed}
+              onClick={() => setClosed((closed) => !closed)}
```

Comment:
This may cause re-renders, better we have that in a constant function wrapped with a useCallback
`const  _setClosed=useCallback(()=>{
   setClosed && setClosed(closed)
})`
and we can use the _setClosed in the handler ` onClick={_setClosed}`

Reply:
Completed, thank you for the useful feedback